### DISCUSSION TITLE
[Enhancement]: improve sorting algorithm

### DISCUSSION THREAD

**Discussion by lance-374** [ciderapp/Cider#643] [Feature Request]
### The feature you would like

Right now, when sorting by artist, it sorts by artist, then track name. Update it to match the official Apple Music apps: sort by album artist, then album, then disc number, then track number. Also make the algorithm ignore articles (a, an, the) at the beginning of albums, titles, etc. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nate-thegrate** [comment]
> when sorting by artist

Wait—this is something you can do in Cider? I'd love to be able to sort a playlist by Artist, but I don't see any option to.
